### PR TITLE
Add new stat history end-point

### PR DESCRIPTION
**Pull request by Mykhailenko**

### CODE REVIEW

File: common/common-client/src/main/java/org/ow2/proactive/utils/console/MBeanInfoViewer.java [hunk 1]
```diff
@@ -138,10 +224,59 @@ public void invoke(final String mbeanNameAsString, final String operationName, f
...
+            case 'k': // 4 hours
+                return 60 * 60 * 4;
+            case 'H': // 8 hours
+                return 60 * 60 * 8;
+            case 'd': // 1 day
+                return 60 * 60 * 24;
+            case 'w': // 1 week
+                return 60 * 60 * 24 * 7;
+            case 'M': // 1 month
+                return 60 * 60 * 24 * 28;
```
Comment (MaelAudren) [inline common/common-client/src/main/java/org/ow2/proactive/utils/console/MBeanInfoViewer.java]: 1 month => 4 weeks

File: rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/SchedulerStateRest.java [hunk 2]
```diff
@@ -3048,6 +3048,18 @@ public String loginWithCredential(@MultipartForm LoginForm multipart)
         return s.getMappedInfo("ProActiveScheduler:name=RuntimeData");
     }
 
+    @Override
+    @GET
+    @Path("stathistory")
+    @Produces("application/json")
+    public String getStatHistory(@HeaderParam("sessionid") String sessionId) throws NotConnectedRestException {
```
Comment (MaelAudren) [inline rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/SchedulerStateRest.java]: Is `throws NotConnectedRestException` required ?

File: common/common-client/src/main/java/org/ow2/proactive/utils/console/MBeanInfoViewer.java (hunk 1 above)
Comment (Mykhailenko) [reply common/common-client/src/main/java/org/ow2/proactive/utils/console/MBeanInfoViewer.java]: oook. it was there before. I would keep it..

File: rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/SchedulerStateRest.java (hunk 2 above)
Comment (Mykhailenko) [reply rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/SchedulerStateRest.java]: yes

**Comment by ShatalovYaroslav** [comment]
retest it please

**Comment by ShatalovYaroslav** [comment]
retest this please

**Pull request merged**